src/KrakenBattle.cpp

```diff
@@ -149,7 +149,7 @@ namespace KrakenBattle {
          case SendType::Error : textType = "<ERROR>"; break;
          case SendType::End : textType = "<END>"; break;
          default:
-            LOG(FATAL) << "Unknown Send::Type: " << static_cast<int>(type);
+            LOG(FATAL) << "Unknown SendType: " << static_cast<int>(type);
```

Comment:
no test exist for this, it was part of the previous refactoring. I consider it tech debt